files/build_templates/sonic_debian_extension.j2

```diff
@@ -359,7 +359,10 @@ sudo chmod 755 $FILESYSTEM_ROOT/usr/bin/restart_service
 
 # Installed smartmontools version should match installed smartmontools in docker-platform-monitor Dockerfile
 # TODO: are mismatching versions fine for bookworm?
-sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install smartmontools
+echo "deb http://deb.debian.org/debian bookworm-backports main" | sudo tee -a $FILESYSTEM_ROOT/etc/apt/sources.list > /dev/null
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get update
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -t bookworm-backports smartmontools=7.4-2~bpo12+1
```

Comment:
last time we updated smartmontools, it brought many dependencies which was hard to manage later.

in this case, what else dependency package it got installed? can you share the log?

Reply:
@lguohan this is for 202311 for comparison, let me share for master (bookworm) 

![image](https://github.com/sonic-net/sonic-buildimage/assets/45705344/34d3dcfc-408a-4a61-adb8-6a197cef5c72)

Reply:
@lguohan on master(bookworm) running 

![image](https://github.com/sonic-net/sonic-buildimage/assets/45705344/d2f4696e-ed32-42fc-8443-d2b961f229e9)

Reply:
can you get the version to see what else we are bringing from backport repo?

Reply:
`root@str3-7060-acs-1:~# apt-cache show smartmontools | grep Depends
Depends: debianutils (>= 2.2), lsb-base (>= 3.2-14), libc6 (>= 2.27), libcap-ng0 (>= 0.7.9), libgcc-s1 (>= 3.0), libselinux1 (>= 3.1~), libstdc++6 (>= 5.2), libsystemd0`

Reply:
@lguohan below is the dependent versions in 202305

![image](https://github.com/sonic-net/sonic-buildimage/assets/45705344/55514316-bd49-4589-af4a-7b54baed016f)